_pkgdown.yml

```diff
@@ -0,0 +1,17 @@
+url: https://mockery.r-lib.org
+
+home:
+  title: Mocking Library for R
+
+authors:
+  Hadley Wickham:
+    href: http://hadley.nz
```

Comment:
Not sure about other authors' websites, or if they would like them to be included here.